install.sh

```diff
@@ -0,0 +1,285 @@
+#!/bin/bash
+# OS Defaults
+OS=$(uname -s) # Gather OS Name
+USER=$(whoami) # Gather USER Name
+
+#OntoUML-Vp-Plugin Defaults
+repo_name="ontouml-vp-plugin"
+repo_url="https://github.com/propilideno/ontouml-vp-plugin/archive/refs/heads/master.zip"
+ontouml_plugin_path="ontouml-vp-plugin-0.5.3"
+
+# =========== Visual Paradigm Defaults ===========
+#App Default Path
+VISUAL_PARADIGM_APP_DIR_WINDOWS="C:\\Program Files\\Visual Paradigm CE 17.0\\" #Windows
+VISUAL_PARADIGM_APP_DIR_MAC="/Applications/Visual Paradigm.app/Contents/Resources/app/" #MacOS
+VISUAL_PARADIGM_APP_DIR_LINUX="/home/$USER/Visual_Paradigm_17.0/" #Linux
+#Plugin Default Path
+VISUAL_PARADIGM_PLUGIN_DIR_WINDOWS="C:\\Users\\$USER\\AppData\\Roaming\\VisualParadigm\\plugins\\" #Windows
+VISUAL_PARADIGM_PLUGIN_DIR_MAC="/Users/$USER/Library/Application Support/VisualParadigm/plugins/" #MacOS
+VISUAL_PARADIGM_PLUGIN_DIR_LINUX="/home/$USER/.config/VisualParadigm/plugins/" #Linux
+
+function read_App_Path(){
```

Comment:
This function does not work on Windows when trying to change the default path of Visual Paradigm. Changing to this is working:

```bash
function read_App_Path(){
    vp_path="$1"
    while true; do
        echo "Visual Paradigm Path: $vp_path"
        read -p "Confirm (y/n)?: " choice
        case "$choice" in
            y|Y ) 
                if [[ -d "$vp_path" ]]; then
                    break
                else
                    printf "<FOLDER NOT FOUND> Type a valid path!\n"
                fi
            ;;
            n|N ) 
                echo "The path to your Visual Paradigm (APP FOLDER) is: "
                read -r vp_path ;;
            * ) printf "Invalid input\n";;
        esac
    done
    echo "Confirmed Path: $vp_path"
    app_dir="$vp_path"
}
```

Reply:
Fixed on [1a15e8c](https://github.com/OntoUML/ontouml-vp-plugin/pull/146/commits/1a15e8c0dbadab4c77a18e0df57a28ff273ec29b)